test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -197,6 +187,46 @@ public void parse_findCommandDuplicateKeys_parsedCorrectly() {
         assertEquals(keySet, result.getKeywords());
     }
 
+
+    /*
+     * Tests for find person by tag command ======================================================================
+     */
+
+    @Test
+    public void parse_findTagCommandInvalidArgs_errorMessage() {
+        // no keywords
+        final String[] inputs = {
+                "findtag",
+                "findtag "
+        };
+        final String resultMessage =
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindTagCommand.MESSAGE_USAGE);
+        parseAndAssertIncorrectWithMessage(resultMessage, inputs);
+    }
+
+    @Test
+    public void parse_findTagCommandValidArgs_parsedCorrectly() {
+        final String[] keywords = { "key1", "key2", "key3" };
+        final Set<String> keySet = new HashSet<>(Arrays.asList(keywords));
+
+        final String input = "findtag " + String.join(" ", keySet);
+        final FindTagCommand result =
+                parseAndAssertCommandType(input, FindTagCommand.class);
+        assertEquals(keySet, result.getKeywords());
+    }
+
+    @Test
+    public void parse_findTagCommandDuplicateKeys_parsedCorrectly() {
+        final String[] keywords = { "key1", "key2", "key3" };
+        final Set<String> keySet = new HashSet<>(Arrays.asList(keywords));
+
+        // duplicate every keyword
+        final String input = "findtag " + String.join(" ", keySet) + " " + String.join(" ", keySet);
+        final FindTagCommand result =
+                parseAndAssertCommandType(input, FindTagCommand.class);
+        assertEquals(keySet, result.getKeywords());
+    }
+
```

Comment:
For this command you can test executing the command as well, but for now I think this is satisfactory.